charts/lakefs/templates/dev-postgresql.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.useDevPostgres }}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: postgres-secret
+  labels:
+    app: postgres
+data:
+  POSTGRES_DB: postgres
+  POSTGRES_USER: lakefs
+  POSTGRES_PASSWORD: lakefs
+  MAX_CONNECTIONS: "1000"
+---
+apiVersion: apps/v1
+kind: Deployment # Create a deployment
+metadata:
+  name: postgres # Set the name of the deployment
+spec:
+  replicas: 1 # Set 3 deployment replicas
```

Comment:
remove the comment + I think for dev one is enough

Reply:
done